### PR TITLE
OBPIH-3768 Improvements to electronic stock card side bar

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy [hunk 1]
```diff
@@ -66,6 +67,15 @@ class InventoryItemController {
             // now populate the rest of the commmand object
             inventoryService.getStockCardCommand(cmd, params)
 
+            List<ProductPackage> packages = []
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy:70]: Let's add this to the getStockCardCommand() method.

Comment (jmiranda) [reply grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy:70]: Actually we should add this logic as a convenience method on Product. 
```
List getUoms() { 
  // do a thing, return a List of Maps or DTOs
  return productPackages.collect { [uom: it.uom, quantity: it.quantity] }.unique()

  // or use one of the unique methods below to do it in the other direction 
  //return productPackages.unique { ... }.collect { [uom: it.uom, quantity: it.quantity ] }
  //return productPackages.unique(new UomComparator()).collect { [uom: it.uom, quantity: it.quantity ] }
}
```

https://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/Collection.html#unique(groovy.lang.Closure)
https://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/Collection.html#unique(java.util.Comparator)

File: grails-app/views/inventoryItem/_productDetails.gsp [hunk 2]
```diff
@@ -28,14 +28,14 @@
                             ${warehouse.message(code:'default.each.label') }
                         </g:else>
                     </div>
-                    <g:if test="${productInstance?.packages }">
-                        <g:each var="productPackage" in="${productInstance?.packages }">
+                    <g:if test="${productInstance?.packages}">
```
Comment (jmiranda) [inline grails-app/views/inventoryItem/_productDetails.gsp:31]: This will just change to productInstance.uoms

**Review by jmiranda — CHANGES_REQUESTED** [review]

**Review by awalkowiak — APPROVED** [review]

**Comment by DominikaHatala** [comment]
@jmiranda I have applied required changes.

**Review by jmiranda — APPROVED** [review]

**Pull request merged**